microsim/opencl/ramp/kernels/ramp_ua.cl

```diff
@@ -302,10 +316,33 @@ kernel void people_update_statuses(uint npeople,
         {
           // Calculate recovered prob based on age
           ushort person_age = people_ages[person_id];
-          float recovery_prob = get_recovery_prob_for_age(person_age, params);
+          float mortality_prob = get_mortality_prob_for_age(person_age, params);
+
+          ushort person_obesity = people_obesity[person_id]; 
+          if (person_obesity > 0){ // if person is obese then adjust mortality probability
+            mortality_prob *= get_obesity_multiplier(person_obesity, params);
+          }
+          
+          // if person has cardiovascular disease then adjust mortality probability
+          ushort person_cvd = people_cvd[person_id];
+          if (person_cvd){
+            mortality_prob *= params->cvd_multiplier;
+          }
+
+          // if person has diabetes then adjust mortality probability       
+          ushort person_diabetes = people_diabetes[person_id];
+          if (person_diabetes){

```

Comment:
To continue on from this, you could probably write this as a single expression like:
```c
float mortality_prob = (float)people_cvd[person_id] * params->cvd_multiplier + 
                       (float)people_diabetes[person_id] * params->diabetes_multiplier + 
                       (float)person_bloodpressure[person_id] * params->bloodpressure_multiplier;
```
Totally optional though.

Reply:
I don't think this logic is quite right, unless I'm missing something here. Since we should be multiplying all these together rather than adding them. I think I'll just stick to the one liner for each health condition, as according to your previous comment.

Reply:
Following on from my above correction, this would now be:
```c
float mortality_prob = (1.0 - (1.0 - params->cvd_multiplier) * (float)people_cvd[person_id]) *
                       (1.0 - (1.0 - params->diabetes_multiplier) * (float)people_diabetes[person_id]) *
                       (1.0 - (1.0 - params->bloodpressure_multiplier) * (float)people_bloodpressure[person_id]);
```